examples/README.md

```diff
@@ -22,17 +22,17 @@ HSET 'user:001' first_name 'John' last_name 'doe' dob '12-JUN-1970'
 HSET 'user:002' first_name 'David' last_name 'Bloom' dob '03-MAR-1981'
 ```
 
-Great! Now its time to checkpoint the container. Lets set necessary environment variables before we proceed. The following variables should work on k3s.
+Great! Now its time to checkpoint the container. Lets set necessary environment variables before we proceed. The following variables should work on GKE.
 
 ```bash
 export CHECKPOINT_CONTAINER=redis \
-export CHECKPOINT_SANDBOX=redis-6b5bcbb6b6-4vmhf \
-export RESTORE_CONTAINER=redis \
-export RESTORE_SANDBOX=redis-6b5bcbb6b6-4vmhf \
+export CHECKPOINT_SANDBOX=redis-6b5bcbb6b6-tdb4p \
```

Comment:
sandbox name for pods deployed through deployment is not fixed this is not work out of the box for almost anyone.

Reply:
yes, user is expected to manually edit this.